Ex_19/index.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+const birthMonth = "December";
+const firstName = "Natali";
+const LastName = "Fox";
```

Comment:
It's a good practice to write constants in all uppercase letters. It's a good way to differentiate them from other variables.
const BIRTHMONTH = "December";
const AGE = 19;

just by looking at code
alert(AGE);
your fellow developer will understand, "Oh that must be a constant", and will be considering all the rules regarding constants.